staticDHCPd/extensions/official/httpdb.py

```diff
@@ -178,24 +180,80 @@ def _lookupMAC(self, mac):
             raise
 
     def _parse_server_response(self, json_data):
-        json_data['domain_name_servers'] = json_data.get('domain_name_servers') or self._name_servers
+        """
+        Parse the response returned from the server, setting defaults
+         as set in config if needed
+
+        :param dictionary json_data: The type of packet being processed.
+        :return :class:`databases.generic.Definition` definition: The associated
+            definition; None if no "lease" is available.
+        """
+        json_data['domain_name_servers'] = json_data.get('domain_name_servers') \
+         or self._name_servers
```

Comment:
nevermind i see that it is uses elsewhere in this file 
i still think this is stupid